compiler/compiler/src/options.rs

```diff
@@ -17,9 +17,11 @@
 // NOTE: If compiler passes are made optional, pass preconditions and invariants may not necessarily hold true.
 
 #[derive(Clone, Default)]
-pub struct OutputOptions {
+pub struct CompilerOptions {
```

Comment:
Are we using all of these flags?
If we are only using for formal verification purposes should they exist on a separate branch?

Reply:
We use them for our test framework, which makes it difficult to isolate to a separate branch.
FWIW I use these options from time to time when I'm writing Leo programs.